app/Http/Controllers/VideoController.php

```diff
@@ -119,6 +119,12 @@ public function store(Request $request)
         $video->category()->associate(Category::findOrFail($request->get('category')));
         $video->hash = $hash;
         $video->save();
+	//for some reason the input is flipped, but works
+	if ($request->get('nsfw') === 'true') {
+	    $video->tag('sfw');
+	    } else {
+	        $video->tag('nsfw');
+	}
```

Comment:
https://dl.dropboxusercontent.com/u/11268312/indentations_alucard.webm
you get which tabs I mean, right?
also notice that the code additions don't line up vertically with the rest of the code.... because of the tabs